### DISCUSSION TITLE
Question: Round down value from ${fs_free(fs)}

### DISCUSSION THREAD

**Discussion by ghost** [brndnmtthws/conky#1932] [General]
Hey there!

Is there a way to format this variable that it will only ever show GiB value without any decimals? (I.e. "41,7GiB free" turns into "41GiB free"

I could possibly write a custom lua function but wanted to check for a more easy way first.

Alternatively:
Force 2 decimals even when ${fs_free(fs)} normally doesn't show them (e.g. "339GiB free" turns into "339,00GiB free"

Thanks!

**Comment by Caellian** [comment]
Yes, there's a [`pad_percents`](https://conky.cc/config_settings#pad_percents) setting. I believe it should pad values as well, if not it's a bug so create a bug report.
